AppCenter/AppCenter/Internals/Sender/MSAppCenterIngestion.m

```diff
@@ -29,7 +28,9 @@ - (id)initWithBaseUrl:(NSString *)baseUrl appSecret:(NSString *)appSecret instal
   return self;
 }
 
-- (void)sendAsync:(NSObject *)data completionHandler:(MSSendAsyncCompletionHandler)handler {
+- (void)sendAsync:(NSObject *)data
```

Comment:
nit - `:` not aligned correctly

Reply:
I think it is because of `completionHandler` is too long.

Reply:
Jae is right, nothing I can do except rename the method which doesn't seem right to me